tech.feliciano.pocket-casts.metainfo.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be upstream. You removed the appdata file in the previous commit, those are the same files. Please make sure to keep this being a patch if the upstream file still exists.

Reply:
Metainfo is not provided in the upstream package releases unfortunately. I could patch the git repository with these changes instead, I already use that source for the upstream icon. 

The package that is shipped doesn't provide the right icon either.

Reply:
Indeed, I'll open some a PR upstream shortly, the desktop file and metainfo could use some sprucing up.